carbon/src/main/java/carbon/beta/AppBarLayout.java

```diff
@@ -296,6 +297,16 @@ public void setRippleDrawable(RippleDrawable newRipple) {
             newRipple.setCallback(this);
             if (newRipple.getStyle() == RippleDrawable.Style.Background) {
                 super.setBackgroundDrawable((Drawable) newRipple);
+            } else if (newRipple.getStyle() == RippleDrawable.Style.Borderless) {
+                if (getParent() != null) {
+                    if (getParent() instanceof RippleView) {
+                        newRipple.setBounds(0, 0, getWidth(), getHeight());
+                    } else {
+                        Log.d(VIEW_LOG_TAG, "Parent should be a Carbon Library Component for Borderless Ripple to Work dynamically!");
```

Comment:
VIEW_LOG_TAG is always equal "View", it would be better to use name of an actual class